torch/utils/data/dataloader.py

```diff
@@ -101,6 +103,9 @@ class DataLoader(object):
                  :ref:`multiprocessing-best-practices` on more details related
                  to multiprocessing in PyTorch.
 
+    .. note:: By default, ``DataLoader`` consumes a RNG state to determine ``base_seed`` 
+              for data-loading threads. 
```

Comment:
Can you instead add the note to https://github.com/pytorch/pytorch/blame/4ba1c4f798fa0aeb00705d8f545168d183393a38/docs/source/data.rst#L336?

Also, the workers are separate processes, not threads.

Reply:
I have moved the note to `data.rst`. Apparently, it was already there in `data.rst` that RNG was used to generate `base_seed`. Just added a side note to it to say its mandatory. Pl let me know if there is any change needed.